CONTRIBUTING.md

```diff
@@ -40,12 +40,26 @@ the PR.
 
 ### Adding an API that utilizes version checks for compatibility
 
+This option should only be used for work that meets ALL of the below requirements:
+
+- Work which has already been discussed with the TeamsJS devs as using this approach
```

Comment:
This is a great addition! Suggestion for some rewording that I think adds clarity:

- Features which have already been discussed with the TeamsJS owners and for which approval to use this approach has been granted,
- Feature implementation that has a requirement of running in host clients that have not onboarded to the new declarative capability support architecture.

Note: typo in existing text ("capabiity")

Reply:
Great suggestions. Will make!